src/AbstractSQLOptimiser.ts

```diff
@@ -1552,17 +1564,8 @@ const typeRules = {
 			],
 		),
 	),
-	Endswith: rewriteMatch(
-		'Endswith',
-		[TextValue, TextValue],
-		Helper<MatchFn<LikeNode>>(
-			([haystack, needle]: [TextTypeNodes, TextTypeNodes]) => [
-				'Like',
-				haystack,
-				['Concatenate', ['EmbeddedText', '%'], ['EscapeForLike', needle]],
-			],
-		),
-	),
+	Endswith: EndsWithMatcher,
+	EndsWith: EndsWithMatcher,
```

Comment:
@Page- this is duplicated, I think you want to remove one

Reply:
This is to support both `['Endswith', ...]` and `['EndsWith', ...]`

Reply:
:facepalm: thanks